graph/rewards_test.go

```diff
@@ -866,3 +866,123 @@ func (r *RewardsQueryTestSuite) Test_Query_GetEarningsByVehicle_BackPaginate_Las
 	  }
 	`, beneficiary.Hex(), currTime.Format(time.RFC3339), beneficiary.Hex(), currTime.Format(time.RFC3339), beneficiary.Hex(), beneficiary.Hex()), string(b))
 }
+
+func (r *RewardsQueryTestSuite) Test_Query_GetAftermarketDeviceEarnings_FwdPaginate() {
+	currTime := time.Now().UTC().Truncate(time.Second)
+	_, beneficiary, err := test.GenerateWallet()
+	r.NoError(err)
+
+	r.createDependencies()
+
+	// Aftermarket Earnings
+	adEarn, ok := new(big.Int).SetString("59147051345528509681", 10)
+	r.NotZero(ok)
```

Comment:
This seems more natural:

https://pkg.go.dev/github.com/stretchr/testify/assert#True

Reply:
I do love these E2E tests.